python/cuml/tests/test_umap.py

```diff
@@ -685,7 +685,6 @@ def test_fuzzy_simplicial_set(n_rows, n_features, n_neighbors):
         "chebyshev",
         "cosine",
         "correlation",
-        "jaccard",
```

Comment:
Why did you remove this metric?

Reply:
It appears that the `jaccard` metric is actually not implemented. I should maybe open an issue for this.

Reply:
Yes, please.

Reply:
Created https://github.com/rapidsai/raft/issues/1647